include/power_grid_model/math_solver/iterative_linear_se_solver.hpp

```diff
@@ -249,30 +269,27 @@ class MeasuredValues {
     // not relevant for the main calculation, as extra data for load_gen/source calculation
     std::vector<SensorCalcParam<sym>> extra_value_;
     // array of partial injection measurement, the bus with not all connected appliances measured
-    std::vector<SensorCalcParam<sym>> partial_injection_;
+    std::vector<SensorCalcParam<sym>> partial_load_injection_;
```

Comment:
why this change?

Partial injection includes load, gen and source injection together, if at least one connected source or load or gen is not measured.